tests/test_apply.py

```diff
@@ -537,6 +538,9 @@ def test_expval_single_wire_no_parameters(
         dev = qubit_device(wires=1)
         obs = operation(wires=[0])
 
+        if device_name == "lightning.gpu" and dev.R_DTYPE == np.float32:
```

Comment:
Is that a fundamental limitation or just for now?

Reply:
Fixed!